AdblockPlusSafariTests/AdblockPlusSafariTests.m

```diff
@@ -90,4 +91,13 @@ - (void)testEasylistPlusExceptionsFilterListsMergeWithWhitelistedWebsites
   [self performMergeFilterlists:@"easylist+exceptionrules_content_blocker"];
 }
 
+- (void)testEasylistFilterListsMergeWithWhitelistedWebsitesV2
+{
+  [self performMergeFilterlists:@"easylist_content_blocker_v2"];}
```

Comment:
Tests for old a new type of filter lists. Note that, files with now format are included just in test target (they are not bundled with the application.